django_app/redbox_app/redbox_core/models.py

```diff
@@ -255,3 +256,23 @@ class ChatMessage(UUIDPrimaryKeyBase, TimeStampedModel):
 
     def __str__(self) -> str:  # pragma: no cover
         return f"{self.text} - {self.role}"
+
+
+class ChatMessageRating(TimeStampedModel):
+    chat_message = models.OneToOneField(ChatMessage, on_delete=models.CASCADE, primary_key=True)
+    rating = models.PositiveIntegerField(validators=[validators.MinValueValidator(1), validators.MaxValueValidator(5)])
```

Comment:
Do we need to account for a future where the range of options changes? Even if we update these validators, will we be able to make sure that ratings gathered when they are out of 5 are comparable to ones gathered when it's out of 10 (for example)?

Reply:
We would need to fiddle with the migrations if the range changes - so, doubling all existing ratings, given our examples. Easy enough to do, I think.